src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -90,26 +108,51 @@ export const ConfirmBidRoute: React.FC<BidProps> = props => {
       errorMessages = [error.message]
     }
 
-    // TODO: add tracking with errorMessages
-    logger.error(errorMessages) // remove once tracking is implemented
+    trackConfirmBidFailed(bidderId, errorMessages)
 
     actions.setSubmitting(false)
     actions.setStatus("submissionFailed")
   }
 
+  function trackConfirmBidFailed(bidderId: string, errors: string[]) {
+    tracking.trackEvent({
+      action_type: Schema.ActionType.ConfirmBidFailed,
+      bidder_id: bidderId,
+      error_messages: errors,
+      ...commonProperties,
+    })
+  }
+
+  function trackConfirmBidSuccess(positionId: string, bidderId: string) {
```

Comment:
minor: thoughts on renaming to `trackConfirmBidSubmitted` for parity between the `action_type` and the function name?